prime-router/src/main/kotlin/serializers/Hl7Serializer.kt

```diff
@@ -736,7 +737,7 @@ class Hl7Serializer(
 
             // Override with organization name if available
             Hl7Configuration.OrderingFacilityName.ORGANIZATION_NAME -> {
-                val organizationName = report.getString(row, "organization_name") ?: rawFacilityName
+                val organizationName = report.getString(row, "organization_name").trimToNull() ?: rawFacilityName
```

Comment:
is the expected behavior to use `rawFacilityName` in the case where the `organization_name` may just be whitespace? e.g. `   ` 

The custom `trimToNull` function doesn't trim out anything, it only returns null if ifEmpty is true.